src/components/legend/draw.js

```diff
@@ -308,7 +308,7 @@ function _draw(gd, legendObj) {
             }
 
             if(gd._context.edits.legendPosition) {
-                var xf, yf, x0, y0;
+                var xf, yf, x0, y0, legendWidth, legendHeight;
```

Comment:
Wondering if we really need these new variables? Can't we use `._width` and `._height` directly?

Reply:
Good point.  Now it's an even smaller change from the original code.